ci-operator/step-registry/cucushift/installer/check/gcp/disktype-compute/cucushift-installer-check-gcp-disktype-compute-commands.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ ! -f "${SHARED_DIR}/compute-osdisk-disktype" ]; then
+  echo "$(date -u --rfc-3339=seconds) - Nothing to do, skip." && exit 0
+fi
+
+CLUSTER_NAME="${NAMESPACE}-${UNIQUE_HASH}"
+
+GOOGLE_PROJECT_ID="$(< ${CLUSTER_PROFILE_DIR}/openshift_gcp_project)"
+export GCP_SHARED_CREDENTIALS_FILE="${CLUSTER_PROFILE_DIR}/gce.json"
+sa_email=$(jq -r .client_email ${GCP_SHARED_CREDENTIALS_FILE})
+if ! gcloud auth list | grep -E "\*\s+${sa_email}"
+then
+  gcloud auth activate-service-account --key-file="${GCP_SHARED_CREDENTIALS_FILE}"
+  gcloud config set project "${GOOGLE_PROJECT_ID}"
+fi
+
+## The expected OS disk type of compute nodes
+expected_disk_type=$(cat "${SHARED_DIR}/compute-osdisk-disktype")
+
+## Try the validation
+ret=0
+
+readarray -t disks < <(gcloud compute disks list --filter="${CLUSTER_NAME}" --format="table(name,type)" | grep worker)
+if [[ ${#disks[@]} == 0 ]]; then
+  echo "$(date -u --rfc-3339=seconds) - Zero compute/worker node found."
```

Comment:
@jianli-wei do you know why https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/openshift_release/41945/rehearse-41945-periodic-ci-openshift-verification-tests-master-installer-rehearse-4.13-installer-rehearse-gcp/1698584378650660864/artifacts/installer-rehearse-gcp/cucushift-installer-check-gcp-disktype-compute/build-log.txt did not find out any worker nodes?